src/components/common/Options/ContextOptions.tsx

```diff
@@ -11,19 +14,35 @@ export default function ContextOptions({
   options = DEFAULT_OPTIONS,
   onClick,
 }: ContextOptionsProps) {
+  const { isVisible, position, closeMenu } = useContextMenuStore();
+
+  useEffect(() => {
+    const handleClickOutside = () => {
+      closeMenu();
+    };
+
+    window.addEventListener('click', handleClickOutside);
+
+    return () => window.removeEventListener('click', handleClickOutside);
+  }, [closeMenu]);
+
+  if (!isVisible) return null;
+
   return (
-    <div className="bg-d10 m-2 border border-d400 rounded-lg">
+    <div
+      className={clsx('absolute m-2 cursor-pointer rounded-lg border border-d400 bg-d10')}
+      style={{ top: position.y, left: position.x }}
+      onClick={() => closeMenu()}
+    >
       {options.map(({ id, name }, idx) => (
         <Fragment key={id}>
           <div
-            className={`px-3 py-1 hover:bg-d30
-            ${idx === 0 && 'rounded-t-lg'}
-            ${idx === options.length - 1 && 'rounded-b-lg'}`}
+            className={`px-3 py-1 hover:bg-d30 ${idx === 0 && 'rounded-t-lg'} ${idx === options.length - 1 && 'rounded-b-lg'}`}
             onClick={() => onClick(id)}
           >
             <p className="font-base text-d400">{name}</p>
           </div>
-          {idx !== options.length - 1 && <div className="border-b border-d400 mx-2"></div>}
+          {idx !== options.length - 1 && <div className="mx-2 border-b border-d400"></div>}
```

Comment:
html 태그에 content가 없으면 요렇게 표현해주세요
```suggestion
          {idx !== options.length - 1 && <div className="mx-2 border-b border-d400" />}
```